File: Java_Programs/While7.java
//wap to accept a no and then print factorial of that no.
import java.util.*;
class While7
{
	public static void main(String []args)
	{
		int num,fact=1;

		Scanner s=new Scanner(System.in);

		System.out.println("Enter a number: ");
		num = s.nextInt();

		while(num>0)
		{
			fact = fact * num;
			num--;
		}
		System.out.println("Factorial = "+ fact);
	}
}

File: C_Program/while7.c
//wap to accept a no. and print factorial of that no.

#include<stdio.h>
int main()
{
	int num,fact=1;
	printf("\n enter the num: ");
	scanf("%d",&num);
	while(num>0)
	{
		fact=fact*num;
		num=num-1;
	}
	printf("\n factorial = %d",fact);
}